package.json

```diff
@@ -247,6 +247,7 @@
     "chai-parentheses": "^0.0.2",
     "chai-string": "^1.5.0",
     "chai-subset": "^1.6.0",
+    "chalk": "^5.2.0",
```

Comment:
We have `kleur` for doing coloured console output, please use this instead.

Reply:
Understood, I noticed we use [chalk in the actions bundle size dist](https://github.com/ipfs/aegir/blob/master/actions/bundle-size/dist/index.js) still, we may want to remove it there as well.